cmd/access/node_builder/access_node_builder.go

```diff
@@ -113,6 +113,8 @@ func DefaultAccessNodeConfig() *AccessNodeConfig {
 			UnsecureGRPCListenAddr:    "0.0.0.0:9000",
 			SecureGRPCListenAddr:      "0.0.0.0:9001",
 			HTTPListenAddr:            "0.0.0.0:8000",
+			RESTListenAddr:            "0.0.0.0:8080",
```

Comment:
the REST server feels like something that should be explicitly enabled (the gRPC-Web as well, but that's out of scope).

Could we simplify the config and only enable the rest server if the address if configured?

Reply:
so I have added a new flag [serve-rest-api](https://github.com/onflow/flow-go/blob/vishal/rest_api_implementation/cmd/access/node_builder/access_node_builder.go#L557) to enable/disable the REST Server.

Reply:
I'm suggesting it should be disabled by default since it's a non-critical interface that not all operators would use. The gRPC-Web one probably should be as well, but that's out of scope and harder to change since it's already enabled by default.

Reply:
switching the flag default to false to disable by default.

Reply:
since it's off by default, could we just use the existence of `--rest-addr` to indicate that the endpoint should be enabled?

Reply:
yup that makes sense